src/core/Akka.Persistence.Query/EventEnvelope.cs

```diff
@@ -20,14 +20,6 @@ namespace Akka.Persistence.Query
     /// </summary>
     public sealed class EventEnvelope : IEquatable<EventEnvelope>
     {
-        /// <summary>
-        /// Initializes a new instance of the <see cref="EventEnvelope"/> class.
-        /// </summary>
-        [Obsolete("For binary compatibility with previous releases")]
-        public EventEnvelope(Offset offset, string persistenceId, long sequenceNr, object @event)
```

Comment:
Obsolete since v1.4.14

Reply:
I'm going to lean towards keeping this one.